utilities/qi-avs-request.vbs

```diff
@@ -161,50 +152,37 @@ const client_DOB_const   		    = 5 '=  Date of Birth MEMB
 const client_ssn_const		        = 6 '=  SSN
 const client_age_const	            = 7 '=  age MEMB
 const client_sex_const			    = 8 '=  client sex
-const addr_eff_date_const	  		= 9 '=	addr_eff_date
-const resi_line_one_const	  		= 10'= 	resi_line_one
-const resi_line_two_const	   		= 11 '= resi_line_two
-const resi_city_const				= 12'= 	resi_city
-const resi_state_const	     		= 13'= 	resi_state
-const resi_zip_const     			= 14'= 	resi_zip
-const resi_county_const     		= 15'= 	resi_county
-const verif_const	  				= 16'= 	verif
-const homeless_const     			= 17 '= homeless
-const ind_reservation_const  		= 18 '= ind_reservation
-const living_sit_const	     		= 19 '= living_sit
-const res_name_const    			= 20'= 	res_name
-const mail_line_one_const    		= 21'= 	mail_line_one
-const mail_line_two_const     		= 22'=  mail_line_two
-const mail_city_const   			= 23'= 	mail_city
-const mail_state_const    			= 24'= 	mail_state
-const mail_zip_const    			= 25'= 	mail_zip
-const phone_numb_one_const    		= 26'= 	phone_numb_one
-const phone_type_one_const    		= 27'= 	phone_type_one
-const phone_numb_two_const     		= 28'= 	phone_numb_two
-const phone_type_two_const  	   	= 29'= 	phone_type_two
-const phone_numb_three_const     	= 30'= 	phone_numb_three
-const phone_type_three_const    	= 31'= 	phone_type_three
+const marital_status_const          = 9 '=  marital status
 
 FOR EACH person IN HH_member_array
+    CALL navigate_to_MAXIS_screen("STAT", "MEMB")
     CALL write_value_and_transmit(person, 20, 76) 'reads the reference number, last name, first name, and THEN puts it into an array YOU HAVENT defined the avs_members_array yet
-    EMReadscreen ref_nbr, 3, 4, 33
-    EMReadscreen last_name, 25, 6, 30
-    EMReadscreen first_name, 12, 6, 63
-    EMReadscreen MEMB_number, 3, 4, 33
+    EMReadscreen ref_nbr, 2, 4, 33
+
     EMReadscreen last_name, 25, 6, 30
+    last_name = trim(replace(last_name, "_", ""))
+
     EMReadscreen first_name, 12, 6, 63
+    first_name = trim(replace(first_name, "_", ""))
+
     EMReadscreen mid_initial, 1, 6, 79
+    mid_initial = replace(mid_initial, "_", "")
+
     EMReadScreen client_DOB, 10, 8, 42
+
     EMReadscreen client_SSN, 11, 7, 42
     If client_ssn = "___ __ ____" then client_ssn = ""
-    last_name = trim(replace(last_name, "_", "")) & " "
-    first_name = trim(replace(first_name, "_", "")) & " "
-    mid_initial = replace(mid_initial, "_", "")
+
     EMReadScreen client_age, 2, 8, 76
```

Comment:
This can sometimes be over 99, and we have to read 3 spaces, otherwise the math will be weird later.

Reply:
Nevermind - we aren't doing anything with the age here - maybe delete this.